README.md

```diff
@@ -103,27 +149,29 @@ Current release info
 | Name | Downloads | Version | Platforms |
 | --- | --- | --- | --- |
 | [![Conda Recipe](https://img.shields.io/badge/recipe-basemap-green.svg)](https://anaconda.org/conda-forge/basemap) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/basemap.svg)](https://anaconda.org/conda-forge/basemap) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/basemap.svg)](https://anaconda.org/conda-forge/basemap) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/basemap.svg)](https://anaconda.org/conda-forge/basemap) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-basemap--data-green.svg)](https://anaconda.org/conda-forge/basemap-data) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/basemap-data.svg)](https://anaconda.org/conda-forge/basemap-data) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/basemap-data.svg)](https://anaconda.org/conda-forge/basemap-data) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/basemap-data.svg)](https://anaconda.org/conda-forge/basemap-data) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-basemap--data--hires-green.svg)](https://anaconda.org/conda-forge/basemap-data-hires) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/basemap-data-hires.svg)](https://anaconda.org/conda-forge/basemap-data-hires) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/basemap-data-hires.svg)](https://anaconda.org/conda-forge/basemap-data-hires) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/basemap-data-hires.svg)](https://anaconda.org/conda-forge/basemap-data-hires) |
 
-Installing basemap
-==================
+Installing basemap-split
+========================
 
-Installing `basemap` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `basemap-split` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
 
 ```
 conda config --add channels conda-forge
 conda config --set channel_priority strict
 ```
 
-Once the `conda-forge` channel has been enabled, `basemap` can be installed with `conda`:
+Once the `conda-forge` channel has been enabled, `basemap, basemap-data, basemap-data-hires` can be installed with `conda`:
```

Comment:
Here just a minor formatting comment, it should be "`basemap`, `basemap-data` and `basemap-data-hires`".